nixos/modules/services/audio/mpd.nix

```diff
@@ -62,26 +64,15 @@ in {
         '';
       };
 
-      network = {
-
-        host = mkOption {
-          default = "any";
-          description = ''
-            This setting sets the address for the daemon to listen on. Careful attention
-            should be paid if this is assigned to anything other then the default, any.
-            This setting can deny access to control of the daemon.
-          '';
-        };
-
-        port = mkOption {
-          default = 6600;
-          description = ''
-            This setting is the TCP port that is desired for the daemon to get assigned
-            to.
-          '';
-        };
-
+      listenStreams = mkOption {
+        type = types.listOf types.str;
+        default = [ "/run/mpd/socket" "6600" ];
+        example = [ "/run/mpd/socket" "19.168.1.32:6600"];
```

Comment:
No, 6600 just means bind to all interfaces, just like the default mpd configuration.